src/query/service/src/interpreters/hook/refresh_hook.rs

```diff
@@ -151,7 +151,7 @@ async fn do_refresh(ctx: Arc<QueryContext>, desc: RefreshDesc) -> Result<()> {
                             ctx_cloned.clone(),
                             *inverted_index_plan,
                         )?;
-                    let mut build_res = refresh_inverted_index_interpreter.execute2().await?;
+                    let mut build_res = refresh_inverted_index_interpreter.build_pipeline().await?;
```

Comment:
Not all interpreters are built as pipelines; some have side effects

Reply:
👌🏻